### ISSUE TITLE
TERA (323370) Failed to launch

### ISSUE THREAD

**Issue by Dazarlin** [ValveSoftware/Proton#795]
On trying to launch it gives the error message:
"The TERA Launcher is unable to load copycub.dll file.
ErrorMessage: Interop load failed
If this problem persists please visit support.enmasse.com/tera for additional help"

System Info: https://gist.github.com/Dazarlin/1fe5b3d929f71f1f3f598c5b99d0c262

EDIT: Decided to move the comment with the log up here for ease of access. Will leave the comment down below too though. Here's the log: 
[steam-323370.log](https://github.com/ValveSoftware/Proton/files/2329262/steam-323370.log)

**Comment by kisak-valve** [comment]
Hello @Dazarlin,please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Additionally, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by Dazarlin** [comment]
Upon a second attempt at a launch I get the same error with a new one afterwards:
"The TERA Launcher is unable to load steamsupport.dll file.
ErrorMessage: User cancelled...
If this problem persists please visit support.enmasse.com/tera for additional help"

**Comment by Dazarlin** [comment]
[steam-323370.log](https://github.com/ValveSoftware/Proton/files/2326100/steam-323370.log)
Here is the log from the PROTON_LOG=1 %command% Launch option.

**Comment by hfluz** [comment]
I also get exactly the same error.

My System Information: https://gist.github.com/hfluz/8dbf40e4b513c7f21d942bd9b2bd4168
Tera Steam log: https://gist.github.com/hfluz/783f1f2638c530d7be0fd814071377f9

**Comment by NalianNalis** [comment]
Tera works just fine on my Laptop with Ubuntu 18.04 LTS,   Intel 6700HQ and Nvidia GTX 960M 4 GB.
Latest Nvidia drivers.  Proton 3.7-5 beta

about 20 fps on max settings and 60+ on lower settings. 
I used to have the same problem with missing dlls but then i erased the game and did a reinstall and it works.

**Comment by BLaDZer** [comment]
```
System:    Host: blad-pc Kernel: 4.18.5-1-MANJARO x86_64 bits: 64 Desktop: Xfce 4.12.4 
           Distro: Manjaro Linux 
CPU:       Topology: 6-Core model: AMD FX-6100 bits: 64 type: MCP L2 cache: 2048 KiB 
           Speed: 2094 MHz min/max: 1400/3300 MHz Core speeds (MHz): 1: 2075 2: 2184 3: 1534 4: 1566 
           5: 1536 6: 1530 
Graphics:  Device-1: AMD Pitcairn PRO [Radeon HD 7850 / R7 265 / R9 270 1024SP] driver: amdgpu 
           v: kernel 
           Display: x11 server: X.Org 1.20.1 driver: amdgpu unloaded: modesetting 
           resolution: 1360x768~60Hz 
           OpenGL: 
           renderer: AMD Radeon HD 7800 Series (PITCAIRN DRM 3.26.0 4.18.5-1-MANJARO LLVM 8.0.0) 
           v: 4.5 Mesa 18.3.0-devel (git-a0f18f2142)
```
Game launched out of box. But crashed frequently. With medium-high video settings game performance are almost playable but on ultra-ultra+ fps will drop a lot.

After the first crash - second occurs almost immediately until steam was reloaded! Not game but steam! Which is strange for me.

![screenshot_2018-09-08_16-47-52](https://user-images.githubusercontent.com/13044587/45254486-11c85300-b38a-11e8-966f-d25bc70ac1eb.png)
[proton_3.7.3_steam-389300.log.zip](https://github.com/ValveSoftware/Proton/files/2363317/proton_3.7.3_steam-389300.log.zip)
[proton_3.7.6_beta_steam-389300.log.zip](https://github.com/ValveSoftware/Proton/files/2363318/proton_3.7.6_beta_steam-389300.log.zip)

**Comment by hfluz** [comment]
@Llorrin I already reinstalled the game a few times (both Ubuntu 18.04 and Arch), but always get that error through Steam. However it also didn't work for me using Lutris, so maybe it's related to my GPU (RX 580).

**Comment by NalianNalis** [comment]
@hfluz this is quite strange to read. i have tried also with the Intel GPU HD530,  prime-select intel, and the game starts without issues, it runs like crap though. 
i use 4.15.0-33-generic kernel.

**Comment by pandymiku** [comment]
I report the same error with Nvidia 720 and Intel I3 on Arch Linux.

**Comment by inverimus** [comment]
I still get this error with proton 3.16-6. My lutris install of TERA works fine, but if I set the runner to proton I get the same crash.

**Comment by MrPaj** [comment]
The game keeps crashing every 30 min or even less if I am in crowded cities. 
Any idea on how to fix it?

I am using nvidia proprietary driver version 418.56 with a GTX 1060 6GB
Proton 4.2-2
Debian testing with kernel 4.19.0-4-amd64 #1 SMP Debian 4.19.28-2

Thank you

**Comment by blackhole89** [comment]
The copycub.dll-related issues appear to be due to a missing certificate (at least in Debian testing). The certificate-adding steps from [this 2014 blog post](http://m-jorgensen.dk/install-tera-online-using-playonlinux/) fixed them for me. (The patcher is currently running, so I don't know if this will be the only roadblock.)

**Comment by Xandaros** [comment]
TERA crashes a LOT with esync enabled. If you disable it, the framerate goes down dramatically, but the crashes stop.
I do have the number of open files rather high:
```
$ ulimit -Hn
1048576
```

Here is a log from one such crash: https://zerobin.net/?a434af6592c265d3#hPCHJbumfdsS+97DFt+1V2hUAB2ws6IMCxSs7g2ryGs=

**Comment by JulianGro** [comment]
Try the newest proton 4.11-1
I didn't try the game before, but for me it just ran out of the box. After pressing play I just needed to close the launcher and the game started immediately after.

**Comment by Xandaros** [comment]
Indeed, with Proton 4.11 it works much better. Makes sense, since it offers a replacement for esync and the problems were clearly esync-related.

**Comment by I-Iawk** [comment]
> Hello @Dazarlin,please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Additionally, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

System Information: https://gist.github.com/I-Iawk/87cfb7792c6b3523ed06968e704f176b
Steam Log: [steam-323370.log](https://github.com/ValveSoftware/Proton/files/3549181/steam-323370.log)

![Screenshot from 2019-08-27 21-27-53](https://user-images.githubusercontent.com/11224011/63826050-09a65200-c912-11e9-897e-fed47eddf0dc.png)

Getting a ```R6106 - not enough space for thread data when attempting to launch the game.```

@kisak-valve new issues cropped up post patch. I hope you can help investigate or advise what is missing...

**Comment by Oreolek** [comment]
Still does random crashes, even with esync off. No FPS drops.

**Comment by hk3wxy** [comment]
NA launcher doesn't work. [crash with proton and wine]
EU (steam version) run only with proton [4-11-8 and 4-11-13]
